pkg/cmd/admin/node/evacuate.go

```diff
@@ -98,26 +98,52 @@ func (e *EvacuateOptions) RunEvacuate(node *kapi.Node) error {
 		return err
 	}
 
+	rss, err := e.Options.Kclient.ReplicaSets(kapi.NamespaceAll).List(kapi.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	dss, err := e.Options.Kclient.DaemonSets(kapi.NamespaceAll).List(kapi.ListOptions{})
```

Comment:
@kargakis I think we should also check the Job Pods as well... @soltysh mentioned "managed-ref" annotation that all managed pod should set (but there is an concern about backward compatibility where old servers might not set that...)